framework/src/modules/chain/bft/finality_manager.js

```diff
@@ -25,6 +26,7 @@ const {
 	BFTInvalidAttributeError,
 } = require('./errors');
 
+const EVENT_NEW_FINALIZED_HEIGHT = 'EVENT_NEW_FINALIZED_HEIGHT';
```

Comment:
I would rather suggest it to change `EVENT_BFT_FINALIZED_HEIGHT_CHANGED` This way its very clear where ever used from which module and entity that even belongs.

Reply:
ok